Formula/mpd.rb

```diff
@@ -1,10 +1,9 @@
 class Mpd < Formula
   desc "Music Player Daemon"
   homepage "https://www.musicpd.org/"
-  url "https://www.musicpd.org/download/mpd/0.23/mpd-0.23.8.tar.xz"
-  sha256 "86bb569bf3b519821f36f6bb5564e484e85d2564411b34b200fe2cd3a04e78cf"
+  url "https://www.musicpd.org/download/mpd/0.23/mpd-0.23.9.tar.xz"
+  sha256 "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
```

Comment:
Looks like the checksum changed. I wonder if it was updated in-place.

Reply:
Replaced, was getting the sha incorrectly on my end.